### PR TITLE
Run tests against 7.2 not 7.3.0-dev

### PR DESCRIPTION
**Pull request by antograssiot**
travis nigthly is using 7.3.0-dev
see https://travis-ci.org/cakephp/cakephp/jobs/261692749#L509 for example

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ php:
   - 7.0
   - 5.6
   - 7.1
-  - nightly
+  - 7.2
```
Comment (HavokInspiration) [inline .travis.yml]: Shouldn't we keep `nightly` alongside with 7.2 ?

Comment (ADmad) [inline .travis.yml]: @HavokInspiration It's too early to start worrying about 7.3 just yet :)

**Comment by ADmad** [comment]
I cherry picked your commit onto branch `php72-fixes`

File: .travis.yml (hunk 1 above)
Comment (HavokInspiration) [inline .travis.yml]: As you folks see fit.

**Pull request closed without merging**